src/scripts/sections/product.js

```diff
@@ -6,217 +6,202 @@
  * @namespace product
  */
 
-import $ from 'jquery';
-import Variants from '@shopify/theme-variants';
+import {getUrlWithVariant, ProductForm} from '@shopify/theme-product-form';
 import {formatMoney} from '@shopify/theme-currency';
 import {register} from '@shopify/theme-sections';
 
+const classes = {
+  hide: 'hide',
+};
+
+const keyboardKeys = {
+  ENTER: 13,
+};
+
 const selectors = {
-  addToCart: '[data-add-to-cart]',
-  addToCartText: '[data-add-to-cart-text]',
+  submitButton: '[data-add-to-cart]',
+  submitButtonText: '[data-add-to-cart-text]',
   comparePrice: '[data-compare-price]',
   comparePriceText: '[data-compare-text]',
-  originalSelectorId: '[data-product-select]',
   priceWrapper: '[data-price-wrapper]',
-  productImageWrapper: '[data-product-image-wrapper]',
-  productFeaturedImage: '[data-product-featured-image]',
-  productJson: '[data-product-json]',
+  imageWrapper: '[data-product-image-wrapper]',
+  visibleImageWrapper: `[data-product-image-wrapper]:not(.${classes.hide})`,
+  imageWrapperById: (id) => `${selectors.imageWrapper}[data-image-id='${id}']`,
+  productForm: '[data-product-form]',
   productPrice: '[data-product-price]',
-  productThumbs: '[data-product-single-thumbnail]',
-  singleOptionSelector: '[data-single-option-selector]',
+  thumbnail: '[data-product-single-thumbnail]',
+  thumbnailById: (id) => `[data-thumbnail-id='${id}']`,
+  thumbnailActive: '[data-product-single-thumbnail][aria-current]',
 };
 
-const cssClasses = {
-  activeThumbnail: 'active-thumbnail',
-  hide: 'hide',
-};
+register('product', {
+  async onLoad() {
```

Comment:
`async / await` is not supported in IE. Is this a concern as we should be supporting IE 11? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function

Reply:
We transpile theme code through Babel. It's just shopify/theme-scripts we're not transpiling to keep their file size and teeny tiny as possible.